.github/instructions/ui-testing.instructions.md

```diff
@@ -0,0 +1,62 @@
+---
+applyTo: "src/Controls/tests/**/*.cs"
```

Comment:
This file should only apply to the following folders 
src/Controls/tests/TestCases.HostApp 
src/Controls/tests/TestCases.iOS.Tests
src/Controls/tests/TestCases.Android.Tests
src/Controls/tests/TestCases.Shared.Tests
src/Controls/tests/TestCases.Mac.Tests
src/Controls/tests/TestCases.WinUI.Tests